index.html

```diff
@@ -16,19 +16,22 @@
     <nav class="main-nav">
       <div id="hamburger"><i class="icon-menu"></i></div>
       <ul>
-        <li>Bio</li>
-        <li>Projects</li>
-        <li>Contact</li>
+        <li><a href="#" class="icon-home tab" data-content="bioInfo">Bio</a></li>
```

Comment:
Actually, I see what you did. The contact link takes you down to that section for whatever page you are on. That is awesome! I am leaving the first comment as a suggestion for you to consider if you want to apply it that way or keep it the way it is! Either way works perfectly!